submissions/edehJohn/html/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
```

Comment:
I see you deleted your first task. You need to revert that.

Reply:
I DIDNT DELETE ANY OF MY TASK, IF I DO BY MISTAKE, THEN HOW DID I DO IT?  AND WHAT I MAY SUPPOSE TO DO TO PREVENT FURTHER DELETING ?  
PLEASE I NEED HELP AM CONFUSED.
THANKS